utils/build.ps1

```diff
@@ -1606,6 +1608,52 @@ function Build-LLVM([Platform]$Platform, $Arch) {
     }
 }
 
+function Build-Sanitizers([Platform]$Platform, $Arch) {
+  $BareTarget = $Arch.LLVMTarget.Replace("$AndroidAPILevel", "")
+  $LLVMDir = "$(Get-TargetProjectBinaryCache $Arch LLVM)\lib\cmake\llvm"
+  $InstallTo = "$($HostArch.ToolchainInstallRoot)\usr\lib\clang\19"
+
+  Build-CMakeProject `
+    -Src $SourceCache\llvm-project\compiler-rt\lib\builtins `
+    -Bin "$(Get-TargetProjectBinaryCache $Arch ClangBuiltins)" `
+    -InstallTo $InstallTo `
+    -Arch $Arch `
+    -Platform $Platform `
+    -UseBuiltCompilers C,CXX `
+    -BuildTargets "install-compiler-rt" `
+    -Defines (@{
+      CMAKE_MT = "mt";
+      CMAKE_SYSTEM_NAME = $Platform.ToString();
+      LLVM_DIR = $LLVMDir;
+      LLVM_ENABLE_PER_TARGET_RUNTIME_DIR = "YES";
+      COMPILER_RT_DEFAULT_TARGET_ONLY = "YES";
+    })
+  
+  Build-CMakeProject `
+    -Src $SourceCache\llvm-project\compiler-rt `
+    -Bin "$(Get-TargetProjectBinaryCache $Arch ClangRuntime)" `
+    -InstallTo $InstallTo `
+    -Arch $Arch `
+    -Platform $Platform `
+    -UseBuiltCompilers C,CXX `
+    -BuildTargets "install-compiler-rt" `
+    -Defines (@{
+      CMAKE_MT = "mt";
+      CMAKE_SYSTEM_NAME = $Platform.ToString();
+      LLVM_DIR = $LLVMDir;
+      LLVM_ENABLE_PER_TARGET_RUNTIME_DIR = "YES";
+      LLVM_RUNTIMES_TARGET = $BareTarget;
+      COMPILER_RT_DEFAULT_TARGET_ONLY = "YES";
+      COMPILER_RT_BUILD_BUILTINS = "NO";
+      COMPILER_RT_BUILD_CRT = "NO";
+      COMPILER_RT_BUILD_LIBFUZZER = "NO";
+      COMPILER_RT_BUILD_ORC = "NO";
+      COMPILER_RT_BUILD_XRAY = "NO";
+      COMPILER_RT_BUILD_PROFILE = "YES";
+      COMPILER_RT_BUILD_SANITIZERS = "YES";
```

Comment:
I could put this into a CMake cache file, but it won't get much simpler

Reply:
Yeah, I can see why it would not simplify much, but, it does make it more obvious that this is static configuration and not logic when you need a last second change to the build.

Reply:
OTOH the structure in https://github.com/swiftlang/swift/tree/main/cmake/caches forces us to add 7 new files. And they all have identical content. (Unless we outsource CMAKE_SYSTEM_NAME, but it won't make that much of a difference.) I guess it should be: `LLVM-$Platform-$($Arch.LLVMName).cmake`